src/locales/__tests__/index.test.ts

```diff
@@ -31,71 +32,100 @@ describe('Locales', () => {
     jest.clearAllMocks();
   });
 
-  // if don't has the locale in AsyncStorage, should set default value ptBR
-  test('caso não possua o locale no AsyncStorage, deve setar o default ptBR', async () => {
+  // if don't has the locale in AsyncStorage, should set default value from 'react-native-loalize'
+  test('caso não possua o locale no AsyncStorage, deve setar o default de react-native-loalize', async () => {
```

Comment:
here is just a grammar mistake, in Portuguese on this case, we use "... do react-native-localize'. And you wrote "loalize" here  and in the above line.